pyccel/ast/utilities.py

```diff
@@ -445,6 +445,10 @@ def collect_loops(block, indices, new_index_name, tmp_vars, language_has_vectors
                 if compatible_operation(*new_vars, language_has_vectors = language_has_vectors):
                     break
 
+            # TODO [NH]: get all indices when adding axis argument to linspace function
+            if isinstance(line.rhs, NumpyLinspace):
+                line.rhs.ind = indices[0]
```

Comment:
Are you sure about this? Is it not a problem in `NumpyLinspace` is inside a for loop?

Reply:
i don't think it's a problem, i tested it and all is working normal since `indices` contains only indices of the created loop foop